GameData/RealismOverhaul/Engine_Configs/RSRM_Config.cfg

```
@@ -271,6 +282,14 @@
 			@name = RSRM-1986
 			@description = Redesigned RSRM, after Challenger disaster
 			@specLevel = operational
+
+			//Redesigned RSRM: 216 flown, 0 failures
+			@TESTFLIGHT
+			{
+				@cycleReliabilityStart = 0.995622
+				@cycleReliabilityEnd = 0.999309
+				%techTransfer = RSRM-1981:50
+			}
 		}
 
 		+CONFIG[RSRM-1981]
```

With the TF data now living inside the config, the correct values are in the '86 config, so better to copy from there.
With the current copy, the techTransfer key doesn't exist, so no edit is made
```suggestion
		+CONFIG[RSRM-1986]
```

---

I'm almost sure that's what I had intended to do. Accepting suggestion; will likely squash later.